.gitconfig

```diff
@@ -1,17 +1,34 @@
 [user]
-	name = Ta-Seen Islam
+	name = Ta-Seen
 	email = taseen00.islam@gmail.com
+
 [core]
 	editor = nvim
-[init]
-	defaultBranch = main
-[pull]
-	ff = only
+	pager = delta
+
 [commit]
 	gpgsign = true
+
+[delta]
+	navigate = true
+	syntax-theme = Kanagawa
```

Comment:
`delta` can use `bat`'s theme settings. Refer to my GitHub comment here:

https://github.com/rebelot/kanagawa.nvim/issues/213#issuecomment-1987536469